rpc/dcrwallet.go

```diff
@@ -181,22 +181,22 @@ func (c *WalletRPC) SetVoteChoice(agenda, choice, ticketHash string) error {
 	return c.Call(c.ctx, "setvotechoice", nil, agenda, choice, ticketHash)
 }
 
-// GetBestBlockHeight uses getbestblock RPC to query the height of the best
+// GetBestBlockHeight uses getblockcount RPC to query the height of the best
 // block known by the dcrwallet instance.
 func (c *WalletRPC) GetBestBlockHeight() (int64, error) {
-	var block dcrdtypes.GetBestBlockResult
-	err := c.Call(c.ctx, "getbestblock", &block)
+	var height int64
+	err := c.Call(c.ctx, "getblockcount", &height)
```

Comment:
Is the switch from getbestblock to getblockcount just for the sake of efficiency? Or is there a functional difference?